midikeyboard/src/main/AndroidManifest.xml

```diff
@@ -1,12 +1,9 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.mobileer.midikeyboard"
-        android:versionCode="4"
-        android:versionName="@string/versionName">
```

Comment:
Well, Gradle can't reference values from strings.xml.
But Gradle can inject string resources, so we can do the reverse.
Having Gradle generate the versionName value and a R.string.version_name for use in the application.
I'll update the pull request.